core/src/main/scala/org/http4s/jdkhttpclient/JdkWSClient.scala

```diff
@@ -44,120 +43,133 @@ import java.util.concurrent.CompletionStage
 object JdkWSClient {
 
   /** Create a new `WSClient` backed by a JDK 11+ http client. */
+  @deprecated("Use `make` which does not return a `Resource`", "0.7.1")
   def apply[F[_]](
       jdkHttpClient: HttpClient
-  )(implicit F: Async[F]): Resource[F, WSClient[F]] = Dispatcher[F].map { dispatcher =>
```

Comment:
Similar change here: instead of sharing a single, global dispatcher for the entire client, we now have an efficient "sequential" dispatcher per-connection.